install_packages.R

```diff
@@ -97,7 +97,17 @@ for(i in seq_along(chapter_pkgs)) {
     message("### CHAPTER: ", i, " ###")
     pkgsAvailable <- installed.packages()[, "Package"]
     pkgsToInstall <- setdiff(chapter_pkgs[[i]], c(pkgsAvailable, pkgs_github))
-    BiocManager::install(pkgsToInstall, update = FALSE, upgrade = FALSE, ask = FALSE, type = pkg_type)
+    #installs the devel versions of mia, miaViz and bluster
+    devel_pkgs <- c("mia", "miaViz", "bluster")
+    if (any(devel_pkgs %in% pkgsToInstall)) {
+        BiocManager::install(devel_pkgs[devel_pkgs %in% pkgsToInstall], update = FALSE, upgrade = FALSE, ask = FALSE, type = pkg_type, version = "devel")
```

Comment:
As suggested in the issue, the packages should be bleeding-edge versions from GH.

- Github contains the most recent changes.

- Bioconductor devel contains versions that have been gone through testing in GH (there is some days delay in updating).

- Bioconductor release contains versions that are udpating 2 times per year based on Bioconductor devel version.

So instead of installing these from Bioconductor devel, install these packages from GH with remotes. Also add comment that the installation step must be removed before Bioc submission